M/MAGEMin/build_tarballs.jl

```diff
@@ -56,25 +43,14 @@ install_license LICENSE
 
 augment_platform_block = """
     using Base.BinaryPlatforms
-    $(MPI.augment)
-    augment_platform!(platform::Platform) = augment_mpi!(platform)
 """
```

Comment:
I don't think you need this at all if you don't want to link to MPI.  Also, remove from it from the last line too.

Reply:
ok thank you very much